### PR TITLE
Update to swagger docs

### LINKED ISSUE #222: Update Swagger to Match Endpoints
# Requirements
- Update the [swagger schema](https://github.com/Bounties-Network/BountiesAPI/blob/master/bounties_api/static/apischema.yaml) to be in line with our current api

# Communication
- [Join our slack community](https://join.slack.com/t/bountiesnetwork/shared_invite/enQtMzA2Mjk3MzAzODQwLTZjN2UxMmU5MWYxZTVmMmM4OGNjZDRiMDgwYTVhOTIwYmQ4MjVlMjNkZjYzOTE4MWI4OTFhOWE4ZTUzN2MyNWY) and message in our community-dev channel for questions
- Put a PR up directly into the bounties-api repo

### PR DESCRIPTION
**Pull request by iamonuwa**
@villanuevawill please review

Fixes #222

### CODE REVIEW

**Comment by codeluggage** [comment]
Looks like CircleCI failed because of something unrelated to this PR. I'll look into that.

File: bounties_api/static/apischema.yaml [hunk 1]
```diff
@@ -228,6 +228,17 @@ paths:
 
 
   /user/{address}/profile/:
+    post:
+      operationId: user_update_profile_by_address
+      description: 'Update user profile by address'
+      parameters: []
```
Comment (villanuevawill) [inline bounties_api/static/apischema.yaml]: @iamonuwa you need to add the parameter for address in here

File: bounties_api/std_bounties/urls.py [hunk 2]
```diff
@@ -15,6 +15,7 @@
 urlpatterns = [
     url(r'^leaderboard/issuer/$', views.LeaderboardIssuer.as_view()),
     url(r'^leaderboard/fulfiller/$', views.LeaderboardFulfiller.as_view()),
+    url(r'^fulfiller/(?P<address>\d+)', views.UserProfile.as_view()),
```
Comment (villanuevawill) [inline bounties_api/std_bounties/urls.py]: @iamonuwa why are you adding a url here? UserProfile already exists as an endpoint. It is in the User app.

Comment (villanuevawill) [reply bounties_api/std_bounties/urls.py]: No need to add this extra endpoint.

File: bounties_api/static/apischema.yaml [hunk 3]
```diff
@@ -210,7 +210,7 @@ paths:
       parameters: []
       responses:
         '200':
-          description: 'Get current user information'
+          description: 'Update current user information'
```
Comment (villanuevawill) [inline bounties_api/static/apischema.yaml]: lets say update user email settings.

File: bounties_api/std_bounties/urls.py (hunk 2 above)
Comment (iamonuwa) [reply bounties_api/std_bounties/urls.py]: Will check again.

File: bounties_api/static/apischema.yaml (hunk 3 above)
Comment (iamonuwa) [reply bounties_api/static/apischema.yaml]: Cool

**Comment by iamonuwa** [comment]
> Looks like CircleCI failed because of something unrelated to this PR. I'll look into that.

Got scared. CircleCI has never failed me.

File: bounties_api/static/apischema.yaml (hunk 1 above)
Comment (iamonuwa) [reply bounties_api/static/apischema.yaml]: Right

**Comment by villanuevawill** [comment]
@iamonuwa is this in prog?

**Comment by iamonuwa** [comment]
Yes, @villanuevawill almost done resolving the comments. Changed my PC on Saturday. Had to setup everything completely today

File: bounties_api/std_bounties/urls.py (hunk 2 above)
Comment (iamonuwa) [reply bounties_api/std_bounties/urls.py]: @villanuevawill I'll remove this as method from bounties views?

**Comment by villanuevawill** [comment]
@iamonuwa this misses the mark pretty far of what was asked for and you didn't respond to the requests. I'm going to close this task out.

**Pull request closed without merging**